src/Sdk/RSWebApi/RunServiceHttpClient.cs

```diff
@@ -9,13 +9,23 @@
 using GitHub.Services.Common;
 using GitHub.Services.OAuth;
 using GitHub.Services.WebApi;
+using Newtonsoft.Json;
```

Comment:
I think we need to do this in `TaskAgentHttpClient` too, to fix the same issue from Actions Service